Content.Server/Coordinates/SpawnRandomOffsetSystem.cs

```diff
@@ -0,0 +1,19 @@
+using Content.Shared.Random.Helpers;
+
+namespace Content.Server.Coordinates;
+
+public sealed class SpawnRandomOffsetSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SpawnRandomOffsetComponent, ComponentInit>(OnInit);
```

Comment:
why not MapInit?

Reply:
>  Because of the way this works, it'll also apply an offset when added to an entity that already exists... But I don't think it's a big issue.

Assuming its unintentional, yeah this should just be map init. If for whatever reason this offset-on-add is desired, it should still use map init + an init function that checks if the entity is already post map init (i.e., got added to an entity after map init).

Reply:
is there a reason why mapinit would be used over component init? Is it a performance thing?

Reply:
Currently every time a mapper loads up an un-initialized map, any entities with this component will randomly shift around.

In general on-init randomization should only ever be done once. So either via the map-init event, or the component just has to disable or remove itself after it has made its random changes.